packages/eslint-plugin-gestalt/src/helpers/styleHelpers.js

```diff
@@ -74,7 +74,7 @@ const getMatchKeyErrorsReducer: GetMatchKeyErrorsReducerType = ({ context }) =>
     };
 
     function includeKey(keyName) {
-      const { onlyKeys } = context.options[0] || {};
+      const { onlyKeys } = context?.options?.[0] ?? {}; // Access options from Esling configuration
```

Comment:
nit: typo "Eslint"

Reply:
Ag I fixed this and I guess some revert of changes put it back. Thanks for catching it.